$NetBSD: patch-aa,v 1.17 2016/08/15 12:26:54 richard Exp $

add quotes when testing with 'which' as, on at least SunOS, it doesn't
return an empty string when the object isn't found.

--- configure.orig	2012-01-19 19:19:29.000000000 +0000
+++ configure
@@ -29366,7 +29366,7 @@ else
   ;;
   *)
   as_save_IFS=$IFS; IFS=$PATH_SEPARATOR
-for as_dir in  /usr/lib/qt-3.3/bin ${QTDIR}/bin${pathlist}
+for as_dir in  /usr/lib/qt-3.3/bin ${QTDIR}/bin ${pathlist}
 do
   IFS=$as_save_IFS
   test -z "$as_dir" && as_dir=.
@@ -29661,9 +29661,9 @@ fi
     fi
 
     KDE3_PLUGINDIR="${KDE3_PREFIX}/lib/kde3"
-    KDE3_SERVICESDIR="${KDE3_PREFIX}/share/services"
-    KDE3_CONFIGDIR="${KDE3_PREFIX}/share/config"
-    KDE3_APPSDATADIR="${KDE3_PREFIX}/share/apps/klash"
+    KDE3_SERVICESDIR="${KDE3_PREFIX}/share/kde/services"
+    KDE3_CONFIGDIR="${KDE3_PREFIX}/share/kde/config"
+    KDE3_APPSDATADIR="${KDE3_PREFIX}/share/kde/apps/klash"
 
   if test x"${ac_cv_path_kde3_incl}" != x ; then
     if test x"${ac_cv_path_kde3_incl}" != x"-I/usr/include"; then
@@ -30855,7 +30855,7 @@ fi
       KDE4_CONFIGDIR="${KDE4_PREFIX}/share/kde4/config"
     fi
     if test x"${KDE4_APPSDATADIR}" = x ; then
-      KDE4_APPSDATADIR="${KDE4_PREFIX}/share/kde4/apps/klash"
+      KDE4_APPSDATADIR="${KDE4_PREFIX}/share/kde/apps/klash"
     fi
 
   if test x"${ac_cv_path_kde4_incl}" != x ; then
@@ -30994,7 +30994,7 @@ $as_echo_n "checking whether NPString ha
 
                   #include "npapi.h"
                   #include "npruntime.h"
-                  int main(int argc, char* argv){
+                  int main(int argc, char** argv){
 	            NPString str;
                     uint32_t len = str.UTF8Length;
 	            return 0;
@@ -31356,6 +31356,9 @@ else
     if test x"${with_x11_incl}" != x ; then
       if test -f ${with_x11_incl}/X11/X.h ; then
        ac_cv_path_x11_incl="-I`(cd ${with_x11_incl}; pwd)`"
+
+$as_echo "#define HAVE_X11_X_H 1" >>confdefs.h
+
       else
        as_fn_error "${with_x11_incl} directory doesn't contain X.h" "$LINENO" 5
       fi
@@ -44135,7 +44138,7 @@ ac_compiler_gnu=$ac_cv_c_compiler_gnu
   dirname=""
   libname=""
       boost_headers="detail/lightweight_mutex.hpp thread/thread.hpp multi_index_container.hpp multi_index/key_extractors.hpp thread/mutex.hpp program_options/options_description.hpp iostreams/stream.hpp"
-      boost_libs="thread program_options iostreams"
+      boost_libs="thread program_options iostreams system"
 
       cygnal_boost_libs="serialization date_time"
 
@@ -50203,7 +50206,7 @@ yast_war="`cat $yast_war`"
 yast_rec="`cat $yast_rec`"
 
 # Pipe stderr to /dev/null since Fedora complains when target isn't there.
-if test x`which apt-get 2>/dev/null` != x; then
+if test x"`which apt-get 2>/dev/null`" != x; then
   if test x"$deb_err" != x -o x"$deb_war" != x -o x"$deb_rec" != x; then
     echo "#!/bin/sh" > deb-attempt-install-dependencies.sh
     echo "packages=\"$deb_err $deb_war $deb_rec\"" >> deb-attempt-install-dependencies.sh
@@ -50216,7 +50219,7 @@ if test x`which apt-get 2>/dev/null` !=
   fi
 fi
 
-if test x`which yum 2>/dev/null` != x; then
+if test x"`which yum 2>/dev/null`" != x; then
   if test x"$rpm_err" != x -o x"$rpm_war" != x -o x"$deb_rec" != x; then
     echo "#!/bin/sh" > rpm-attempt-install-dependencies.sh
     echo "packages=\"$rpm_err $rpm_war  $rpm_rec\"" >> rpm-attempt-install-dependencies.sh
